### ISSUE TITLE
packer_cleanup

### ISSUE THREAD

**Issue by bkblain** [andrewrothstein/ansible-packer#1]
The new cleanup task is failing with undefined on ``packer_cleanup``. I guess you might have forgotten to check in a file.

/defaults/main.yml   ``packer_clean=False``
/tasks/main.yml       ``when: packer_cleanup``

**Comment by andrewrothstein** [comment]
I think I fixed that in the last commit. Can u forcefully reinstall the latest. Perhaps I forgot to publish the master to the galaxy?

**Comment by bkblain** [comment]
I am looking at the code in github. Unless I'm missing something, the variables don't match. I'll try running it again later tonight.

edit: Adding logs

```
- downloading role 'packer', owned by andrewrothstein
- downloading role from https://github.com/andrewrothstein/ansible-packer/archive/v2.0.9.tar.gz
- extracting andrewrothstein.packer to /myproject/ansible/roles/andrewrothstein.packer
- andrewrothstein.packer (v2.0.9) was installed successfully
- adding dependency: andrewrothstein.unarchive-deps (v1.0.8)
```

```
TASK [andrewrothstein.packer : downloading packer...] **********************************
ok: [localhost]

TASK [andrewrothstein.packer : create target packer dir...] ****************************
ok: [localhost]

TASK [andrewrothstein.packer : unarchive...] *******************************************
skipping: [localhost]

TASK [andrewrothstein.packer : linking...] *********************************************
ok: [localhost]

TASK [andrewrothstein.packer : cleanup...] *********************************************
fatal: [localhost]: FAILED! => {"failed": true, "msg": "The conditional check
'packer_cleanup' failed. The error was: error while evaluating conditional
(packer_cleanup): 'packer_cleanup' is undefined The error appears to have been in
'/myproject/ansible/roles/andrewrothstein.packer/tasks/main.yml': line 36, column 3, but
maybe elsewhere in the file depending on the exact syntax problem.The offending
line appears to be:- name: cleanup...  ^ here"}

PLAY RECAP *****************************************************************************
localhost                  : ok=36   changed=0    unreachable=0    failed=1   
```

**Comment by andrewrothstein** [comment]
Late night coding typos! I just published the fix as v2.0.10 to the galaxy.
